operator/v1/types_ingress.go

```diff
@@ -633,8 +636,18 @@ type AWSClassicLoadBalancerParameters struct {
 }
 
 // AWSNetworkLoadBalancerParameters holds configuration parameters for an
-// AWS Network load balancer.
+// AWS Network load balancer. For Example: Setting AWS EIPs https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html
 type AWSNetworkLoadBalancerParameters struct {
+	// eipAllocations assign Elastic IP addresses to the Network Load Balancer.
+	// The number of Allocation IDs must match the number of subnets that are used for the load balancer.
+	// See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html for general
+	// information about configuration, characteristics, and limitations of Elastic IP addresses.
```

Comment:
You should list here any relevant restrictions:

- eipAllocations cannot be used with "Internal" scope.
- An EIP can be allocated to only a single IngressController.
- Considerations around using `eipAllocations` together with `subnets` (see #1841; you might need to work with @gcs278 on this, and come to think of it, #1841 likewise should mention if there are issues with using `eipAllocations` together with `subnets`).
- Any others?